Textile/Classes/Public/TextileApi.m

```diff
@@ -142,34 +147,40 @@ - (Summary *)summary:(NSError * _Nullable __autoreleasing *)error {
   return [[Summary alloc] initWithData:data error:error];
 }
 
+- (BOOL)start:(NSError *__autoreleasing  _Nullable *)error {
+  return [self.lifecycleManager start:error];
```

Comment:
We delegate calls to start and stop to the `TTELifecycleManager` implementation (which can theoretically be different depending on the context)